Source/WebInspectorUI/UserInterface/Views/ColorPicker.js

```diff
@@ -87,7 +87,26 @@ WI.ColorPicker = class ColorPicker extends WI.Object
         this._opacity = 0;
         this._opacityPattern = "url(Images/Checkers.svg)";
 
-        this._color = WI.Color.fromString("white");
+        this.color = WI.Color.fromString("white");
+
+        if (colorVariables) {
```

Comment:
i think this should be `colorVariables?.length` in case there are no variables on the page, as we dont want to show this section unless there's actually something to show in it